modules/default/calendar/calendar.js

```diff
@@ -86,6 +86,9 @@ Module.register("calendar", {
 
 	// Override start method.
 	start: function () {
+		const ONE_SECOND = 1000; // 1,000 milliseconds
+		const ONE_MINUTE = ONE_SECOND * 60;
```

Comment:
const ONE_MINUTE = 60000;

is clear enough :-)

Reply:
This was in line with the rest of the file, I don't mind.